tables/table.py

```diff
@@ -1631,8 +1693,12 @@ def _check_sortby_csi(self, sortby, checkCSI):
                 "Field `%s` must have associated a 'full' index "
                 "in table `%s`." % (sortby, self))
 
-    def itersorted(self, sortby, checkCSI=False,
-                   start=None, stop=None, step=None):
+    def itersorted(self,
+                   sortby: Union["Column", str],
+                   checkCSI: bool=False,
+                   start: Optional[int]=None,
+                   stop: Optional[int]=None,
+                   step: Optional[int]=None) -> Iterator:
```

Comment:
See the comment on `__iter__`

Reply:
As recommended changed to `Iterator[tableextension.Row]`.